authors/apps/authentication/serializers.py

```diff
@@ -13,7 +21,19 @@ class RegistrationSerializer(serializers.ModelSerializer):
     password = serializers.CharField(
         max_length=128,
         min_length=8,
-        write_only=True
+        write_only=True, validators=[password_validator]
+    )
+    email = serializers.EmailField(
+                                   max_length=30,
+                                   validators=[UniqueValidator(
+                                    queryset=User.objects.all(),
+                                    message="Email already exists, please login instead")],
```

Comment:
Same here. I think saying Email already exists is enough.

Reply:
No Lenny, as I researched, I discovered that users need to know what action to take in case of an error. That's what I explain in the next line.